schema.sql/schema

```diff
@@ -0,0 +1,69 @@
+-- Create Database
+CREATE DATABASE catalog_of_things;
+DROP IF EXISTS music_albums, item, genres, books, labels, sources, authors, games;
+
+CREATE TABLE item(
+    id SERIAL PRIMARY KEY,
+    genre_id INT,
+    author_id INT,
+    source_id INT,
+    label_id INT,
+    publish_date DATE,
+    archived BOOLEAN,
+    FOREIGN KEY (genre_id) REFERENCES genre(id),
+    FOREIGN KEY (author_id) REFERENCES author(id),
+    FOREIGN KEY (label_id) REFERENCES label(id)
+    )
```

Comment:
- [ ] [Optional] You don't need to have a column that links `source_id` since the team is made up of three people. Kindly remove this column and the `Sources` table.